docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -101,6 +101,44 @@ Maximum number of concurrent calls to the rendering service.
 
 Scale factor for rendering images. Value `2` is enough for monitor resolutions, `4` would be better for printed material. Setting a higher value affects performance and memory.
 
+## [auditing]
+
+### enabled
+
+Enable the auditing feature
```

Comment:
Put periods at the end of the sentence fragments used as descriptions to maintain consistency.